<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMenu</class>
 <widget class="QDialog" name="UserMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>481</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="contacts">
    <widget class="QListWidget" name="contact_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>481</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>contacts</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="addContactLinEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>341</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>add contact</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addcontact_btn">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>add</string>
     </property>
    </widget>
    <widget class="QLabel" name="add_status">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>status of add</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="msgs">
    <widget class="QListWidget" name="msg_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>481</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="favoriteButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>add to fav</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="favmsg">
    <widget class="QListWidget" name="fav_msg_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="send_msg">
    <widget class="QPushButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>360</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>send msg</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="is_annon">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="text">
      <string>annonemes</string>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>your msg</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>kamel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>wade3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mahmoud</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>choose contact</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MsgSendBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>451</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>431</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>contacts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="msg_tab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>msgs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutButtent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>logout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fav_tab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>fav msgs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_msg_tab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>send msg</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
